Library/Homebrew/cmd/outdated.rb

```diff
@@ -41,7 +41,9 @@ def print_outdated(formulae)
     verbose = ($stdout.tty? || ARGV.verbose?) && !ARGV.flag?("--quiet")
     fetch_head = ARGV.fetch_head?
 
-    outdated_formulae = formulae.select { |f| f.outdated?(fetch_head: fetch_head) }
+    outdated_formulae = formulae
+                        .select { |f| f.outdated?(fetch_head: fetch_head) }
+                        .sort
```

Comment:
Leave the first line as it was, and align `.sort` with `.select`.

Reply:
Fixed in dougal/brew@c80842449